pkg/webhook/client.go

```
@@ -81,3 +82,15 @@ func (si *SidecarInjector) GetPVC(namespace, name string) (*corev1.PersistentVol
 
 	return pvc, nil
 }
+
+// GetVolumesStorageClass returns nil if pv has no sc, error if its unable to retrieve the sc, or for happy path returns the sc
+func (si *SidecarInjector) GetVolumesStorageClass(volume *corev1.PersistentVolume) (*storagev1.StorageClass, error) {
+	if volume == nil || volume.Spec.StorageClassName == "" {
```

If the customer specified `gke-gcsfuse/profies: true` annotation in their Pod spec, then you can assume that there is an explicit intent to use the feature, therefore, having a `""` StorageClassName in their PV would be an invalid user error. It would be better to return an error here (but make sure that this function is only used by gcsfuse profiles).